pyoverkiz/client.py

```diff
@@ -210,18 +211,21 @@ async def somfy_tahoma_get_access_token(self) -> str:
 
             return self._access_token
 
-    async def somfy_tahoma_refresh_token(self, refresh_token: str) -> str:
+    async def refresh_token(self) -> None:
```

Comment:
We don’t need to give in parameter the refresh_token, we can directly use the inner one.

Reply:
Why the rename? I didn't like the previous one as well, however this only applies to Somfy TaHoma (Europe).

Reply:
It’s with the idea to have something generic, and also avoid to test each time the server type before calling this method.